backend/src/main/kotlin/com/group7/artshare/entity/Comment.kt

```diff
@@ -0,0 +1,25 @@
+package com.group7.artshare.entity
+
+import lombok.Data
+import java.util.*
+import javax.persistence.*
+
+@Data
+@Entity
+class Comment {
```

Comment:
When I try to run this branch, it gave me an error related with this class.
Adding the line below solves the error.
@Table(name="comment")